packages/framework/src/Component/Cron/CronFacade.php

```diff
@@ -11,6 +12,9 @@
 
 class CronFacade
 {
+    /**
+     * @deprecated
```

Comment:
Can you please add info about what to use instead of this and add an upgrade note?

Reply:
I have added some info